parking_permits/resolvers.py

```diff
@@ -83,7 +84,9 @@
 def is_valid_address(address):
     if not address:
         return False
-    return address.get("city").upper() == "HELSINKI"
+    if settings.HELSINKI_ADDRESS_CHECK:
```

Comment:
Would it be possible to have a single util function that can be shared between resolvers e.g. `is_valid_city()`?

Also can use `casefold()` for Unicode-friendly case-insensitive check i.e. 

```python

def is_valid_city(city):
     if settings.HELSINKI_ADDRESS_CHECK:
         return city and city.casefold() == "helsinki"
     return True
```